app/assets/javascripts/components/shared/modal.js

```diff
@@ -24,6 +24,11 @@
         showTitle: false,
         // Title of the modal – mandatory for accessibility
         title: 'Untitled modal',
+        // Allows the modal container have scroll
+        allowScroll: false,
```

Comment:
It's not. That's why we have `this.body.classList.toggle('_no-scroll', !this.options.allowScroll);` (previously it was just `this.body.classList.add('_no-scroll')`)